### PR TITLE
Fixes #5998: add taxonomy in config_templates

### PR DESCRIPTION
**Pull request by gulicm**
Config templates cannot be assigned to a taxonomy from config_template#edit
http://projects.theforeman.org/issues/5998

Added taxonomy in Provisioning Templates in edit template.

### CODE REVIEW

File: db/migrate/20090714132448_create_hosts.rb [hunk 1]
```diff
@@ -51,6 +51,7 @@ def self.up
     add_column :hosts, :sp_subnet_id, :integer
     add_column :hosts, :ptable_id, :integer
     add_column :hosts, :medium_id, :integer
+    add_column :hosts, :config_template_id, :integer
```
Comment (dLobatog) [inline db/migrate/20090714132448_create_hosts.rb:54]: Why is this necessary? Hosts use config templates through its operating system

**Comment by domcleal** [comment]
Thanks for raising the PR @gulicm.  It looks like this has been fixed via #2181 in the meantime, which will be in Foreman 1.8, so I'll close this one.
